test/integration/chai/chai-policy-tests.cxx

```diff
@@ -13,14 +13,25 @@ static_assert(RAJA::detail::get_space<RAJA::omp_parallel_for_exec>::value == cha
 
 #if defined(RAJA_ENABLE_CUDA)
 static_assert(RAJA::detail::get_space<RAJA::cuda_exec<128> >::value == chai::GPU, "");
+#endif
+
+#if defined(RAJA_ENABLE_CUDA)
 static_assert(RAJA::detail::get_space<RAJA::IndexSet::ExecPolicy<RAJA::seq_segit, RAJA::cuda_exec<128> > >::value == chai::GPU, "");
```

Comment:
That's fine. I didn't look too hard at this, but was curious if we were making limiting assumptions.